native_client/deepspeech.cc

```diff
@@ -483,7 +631,11 @@ DS_SpeechToText(ModelState* aCtx,
 {
   StreamingState* ctx;
   int status = DS_SetupStream(aCtx, 0, aSampleRate, &ctx);
+#ifndef USE_TFLITE
   if (status != tensorflow::error::OK) {
+#else // USE_TFLITE
+  if (status != kTfLiteOk) {
```

Comment:
Maybe we should just define our own error codes and avoid that kind of stuff ?

Reply:
Yes, I agree.

Reply:
Either way works for me, it'd make this PR look cleaner but it's not a huge deal.

Reply:
https://github.com/mozilla/DeepSpeech/issues/1782